### DISCUSSION TITLE
When loading models attempt to reuse previous model dictionary?

### DISCUSSION THREAD

**Discussion by mark-wd** [vladmandic/sdnext#1003] [General]
Do we know if "When loading models attempt to reuse previous model dictionary" has any disadvantages? Should it just be enabled to see if there's a match?

**Comment by vladmandic** [comment]
- advantage: slightly faster, by 0.1-0.2sec
- disadvantage: sometimes it goes wrong when switching from sd1.5 to sd2.1 model plus there is a chance of memory leak that is impossible to trace.

**Comment by mark-wd** [comment]
Seems like very tangential benefits for a lot of potential trouble.
